torchensemble/voting.py

```diff
@@ -92,17 +92,52 @@ def _parallel_fit_per_epoch(
     """Implementation on the VotingClassifier.""", "model"
 )
 class VotingClassifier(BaseClassifier):
+    def __init__(self, voting_strategy="soft", **kwargs):
+        super(VotingClassifier, self).__init__(**kwargs)
+
+        self._implemented_voting_strategies = ["soft", "hard"]
```

Comment:
This attribute is redundant, maybe we could hard-code the supported strategies, for example, change the attribute in line 100 into a set `("soft", "hard")`, and replace the latter `{}` in line 103 with a fixed string.

Reply:
Check if my latest changes is what you meant :)